recipes/triqs/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "2.2.1" %}
+
+package:
+  name: triqs
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/triqs/releases/download/{{ version }}/triqs-{{ version }}.tar.gz
+  sha256: 523d60a98af0c07e972338f3a5f07d7ea1232d6e67b0f063e139d986f09b9864
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - clangxx >=8  # [osx]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - boost-cpp
+    - fftw
+    - gmp
+    - hdf5
+    - h5py
+    - mako
+    - mpi4py
+    - numpy  1.14
+    - openmpi
+    - python
+    - scipy
+    - libblas
+    - liblapack
+    - libclang
+    - zlib
+  run:
+    - boost-cpp
```

Comment:
Is this needed at runtime?

Reply:
Yes, we are making use of `libboost_serialization` at runtime.